src/rust/src/lazy/dsl.rs

```diff
@@ -20,7 +20,6 @@ use polars::prelude as pl;
 use polars::prelude::{ExprEvalExtension, NestedType, SortOptions};
 use std::any::Any;
 use std::ops::{Add, Div, Mul, Rem, Sub};
-use std::result::Result;
```

Comment:
Why this change? Isn't it cleaner to keep this rather than putting `std::result::` everywhere?